straxen/rundb.py

```diff
@@ -153,10 +153,17 @@ def _find(self, key: strax.DataKey,
                 doc = self.collection.find_one(run_query, projection={'_id'})
                 if not doc:
                     raise ValueError(f"Attempt to register new data for non-existing run {key.run_id}")   # noqa
+
+                # Count the number of files in the output path
+                file_count = None
+                if os.path.exists(output_path) and len(os.listdir(output_path)):
```

Comment:
What's the logic behind `and len(os.listdir(output_path))`? If the folder exists but is empty, wouldn't we want to store `0` rather than `None`?

Reply:
Good point changed it